python/cuml/tests/dask/test_dask_logistic_regression.py

```diff
@@ -305,7 +339,12 @@ def imp():
         datatype, nrows, ncols, n_info, n_classes=n_classes
     )
 
-    X_df, y_df = _prep_training_data(client, X, y, n_parts)
+    if convert_to_sparse is False:
+        # X_dask and y_dask are dask cudf
+        X_dask, y_dask = _prep_training_data(client, X, y, n_parts)
+    else:
+        # X_dask and y_dask are dask array
+        X_dask, y_dask = _prep_training_data_sparse(client, X, y, n_parts)
```

Comment:
Please flip this to `if convert_to_sparse: ... else: ...`.

Reply:
Done.